### ISSUE TITLE
Switching from Networks on the Browser Menu Deactivates all Previously Enabled Networks

### ISSUE THREAD

**Issue by Hoanib** [AlphaWallet/alpha-wallet-android#2603]
When you switch from networks on the menu of the browser that action deactivates all previously activated networks.

**Comment by Devjohn12** [comment]
Talk to their live support about your issues.

**Comment by JamesSmartCell** [comment]
This is the network select in the dapp browser? Can you give a quick example of the issue?

**Comment by Hoanib** [comment]
Yes it's the network select on the Browser. When you switch for another network all the previously enabled networks at Select Active Networks in Settings will be disconnected.
